.github/workflows/ccip-live-network-tests.yml

```diff
@@ -0,0 +1,306 @@
+          cl_image_tag: ${{ env.CHAINLINK_VERSION }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+          go_mod_path: ./integration-tests/go.mod
+          QA_AWS_REGION: ${{ secrets.QA_AWS_REGION }}
+          QA_AWS_ROLE_TO_ASSUME: ${{ secrets.QA_AWS_ROLE_TO_ASSUME }}
+          QA_KUBECONFIG: ${{ secrets.QA_KUBECONFIG }}
+          triggered_by: ${{ env.TEST_TRIGGERED_BY }}
+          artifacts_location: ./integration-tests/load/logs/payload_ccip.json
+          aws_registries: ${{ secrets.QA_AWS_ACCOUNT_NUMBER }}
+          cache_key_id: ccip-load-${{ env.MOD_CACHE_VERSION }}
+          cache_restore_only: "true"
+          should_cleanup: false
+
+  ccip-smoke-test:
+    name: CCIP smoke Test
+    environment: integration
+    runs-on: ubuntu-latest
+    needs: [ build-chainlink, build-test-image ]
+    # if the event is a scheduled event or the test type is load and no previous job failed
+    if: ${{ github.event_name == 'workflow_dispatch' && inputs.test_type == 'smoke' && !contains(needs.*.result, 'failure') }}
+    permissions:
+      issues: read
+      checks: write
+      pull-requests: write
+      id-token: write
+      contents: read
+    env:
+      CHAINLINK_ENV_USER: ${{ github.actor }}
+      SLACK_API_KEY: ${{ secrets.QA_SLACK_API_KEY }}
+      SLACK_CHANNEL: ${{ secrets.QA_SLACK_CHANNEL }}
+      TEST_LOG_LEVEL: info
+      REF_NAME: ${{ github.head_ref || github.ref_name }}
+      ENV_JOB_IMAGE_BASE: ${{ secrets.QA_AWS_ACCOUNT_NUMBER }}.dkr.ecr.${{ secrets.QA_AWS_REGION }}.amazonaws.com/chainlink-ccip-tests
+      BASE64_NETWORK_CONFIG: ${{ secrets.BASE64_NETWORK_CONFIG }}
+
+    steps:
+      - name: Collect Metrics
+        id: collect-gha-metrics
+        uses: smartcontractkit/push-gha-metrics-action@dea9b546553cb4ca936607c2267a09c004e4ab3f # v3.0.0
+        with:
+          id: ccip-on-demand-live-testnet-tests-smoke-test
+          org-id: ${{ secrets.GRAFANA_INTERNAL_TENANT_ID }}
+          basic-auth: ${{ secrets.GRAFANA_INTERNAL_BASIC_AUTH }}
+          hostname: ${{ secrets.GRAFANA_INTERNAL_HOST }}
+          this-job-name: CCIP Smoke Test
+        continue-on-error: true
+      - name: Checkout the repo
+        uses: actions/checkout@9bb56186c3b09b4f86b1c65136769dd318469633 # v4.1.2
+        with:
+          ref: ${{ env.REF_NAME }}
+      - name: Prepare Base64 TOML override
+        shell: bash
+        run: |
+          SLACK_USER=$(jq -r '.inputs.slackMemberID' $GITHUB_EVENT_PATH)
+          echo ::add-mask::$SLACK_USER
+          echo "SLACK_USER=$SLACK_USER" >> "$GITHUB_ENV"
+          if [[ "${{ github.event_name }}" == "workflow_dispatch" ]]; then
+            BASE64_CCIP_CONFIG_OVERRIDE=$(jq -r '.inputs.base64_test_input' $GITHUB_EVENT_PATH)
+            echo ::add-mask::$BASE64_CCIP_CONFIG_OVERRIDE
+            echo "BASE64_CCIP_CONFIG_OVERRIDE=$BASE64_CCIP_CONFIG_OVERRIDE" >> $GITHUB_ENV
+            echo "TEST_BASE64_CCIP_CONFIG_OVERRIDE=$BASE64_CCIP_CONFIG_OVERRIDE" >> $GITHUB_ENV
```

Comment:
curious, what's the difference between `BASE64_CCIP_CONFIG_OVERRIDE` and `TEST_BASE64_CCIP_CONFIG_OVERRIDE`?